index.js

```diff
@@ -1,9 +1,20 @@
-var crypto = require('crypto')
+var native = require('crypto')
+
+function nativePBKDF2 (password, salt, iterations, keylen, digest, callback) {
+  return native.pbkdf2(password, salt, iterations, keylen, digest || 'sha1', callback)
+}
+
+function nativePBKDF2Sync (password, salt, iterations, keylen, digest) {
+  return native.pbkdf2Sync(password, salt, iterations, keylen, digest || 'sha1')
+}
+
 /* istanbul ignore next */
-if (crypto && (!crypto.pbkdf2Sync || crypto.pbkdf2Sync.toString().indexOf('keylen, digest') === -1)) {
-  exports.pbkdf2 = require('./lib/async')
-  exports.pbkdf2Sync = require('./lib/sync')
+if (!native.pbkdf2Sync || native.pbkdf2Sync.toString().indexOf('keylen, digest') === -1) {
+  module.exports.pbkdf2Sync = require('./lib/sync')
```

Comment:
why did you add module. here ?

Reply:
habit of syntax highlighting,  will remove if needed

Reply:
lets not add unneded churn